composer.json

```diff
@@ -33,6 +33,7 @@
         "symfony/twig-bridge":   "~2.5",
         "org_heigl/daterange":   "~1.1",
         "guzzlehttp/guzzle":     "^6.2.1",
-        "monolog/monolog":       "^1.21"
+        "monolog/monolog":       "^1.21",
+        "twbs/bootstrap-sass": "^3.3"
```

Comment:
as npm is already available and used for the jslinting I'd say we should definitely use npm to include frontend-stuff. So adding bootstrap via ```npm install bootstrap-sass -save``` should be ok…